### PR TITLE
Update get started to 2.1

### LINKED ISSUE #6583: Update Get Started node for 2.1 release
Currently the [ASP.NET Core Get Started page for 2.1 RC1](https://docs.microsoft.com/en-us/aspnet/core/getting-started?view=aspnetcore-2.1) says to install ".NET Core SDK 2.0". We need to make sure this page (and any other places we have install steps or list prereqs) is updated for the 2.1 release.

`edit by` @rick-anderson 
See related #5716

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #6583
[Internal review url](https://review.docs.microsoft.com/en-us/aspnet/core/getting-started?view=aspnetcore-2.1&branch=pr-en-us-6764)

### CODE REVIEW

File: aspnetcore/getting-started.md [hunk 1]
```diff
@@ -25,20 +25,34 @@ uid: getting-started
...
-3. Run the app with the following commands:
+3. Trust the HTTPS development certificate:
+
+    ```terminal
+    dotnet dev-certs https --trust
+    ```
+
+    The preceding command displays the following dialog:
+
+    ![Security warning dialog](getting-started/_static/cert.png)
```
Comment (danroth27) [inline aspnetcore/getting-started.md]: This dialog is windows experience. On Mac the experience is different.

File: aspnetcore/getting-started.md [hunk 2]
```diff
@@ -25,20 +25,34 @@ uid: getting-started
     dotnet new webapp -o aspnetcoreapp
     ```
 
-3. Run the app with the following commands:
+3. Trust the HTTPS development certificate:
+
+    ```terminal
+    dotnet dev-certs https --trust
```
Comment (danroth27) [inline aspnetcore/getting-started.md]: Note this only works on Windows and macOS. On Linux you have to manually do distro specific steps to trust the certificate.

**Comment by Rick-Anderson** [comment]
@scottaddie  it's not building the preview - should I open an issue?

**Comment by scottaddie** [comment]
@Rick-Anderson It's unstuck now.

**Comment by Rick-Anderson** [comment]
@danroth27  ready to review. Much cleaner now going with @scottaddie  suggestion of OS tabs.

**Comment by Rick-Anderson** [comment]
@danroth27 It's ready for your review.

File: aspnetcore/getting-started.md [hunk 3]
```diff
@@ -17,28 +17,61 @@ uid: getting-started
...
+    *Trusting the HTTPS development certificate was requested. If the certificate is not already trusted we will run the following command:*
+    `'sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain <<certificate>>'`
+    *This command might prompt you for your password to install the certificate on the system keychain.
+    Password:*
+
+    Enter your password if you agree to trust the development certificate.
+
+# [Linux](#tab/linux)
+
+    See the Linux platform documentation.
```
Comment (danroth27) [inline aspnetcore/getting-started.md]: How about this: "See the documentation for your Linux distribution on how to trust the HTTPS development certificate"

**Pull request merged**